src/dashboard/tests/test_api.py

```diff
@@ -481,6 +481,16 @@ def setUp(self):
         if not os.path.exists(self.config_path):
             os.makedirs(self.config_path)
         install_builtin_config("default")
+        install_builtin_config("automated")
+
+    def test_list_processing_configs(self):
+        response = self.client.get(reverse(views.processing_configurations))
+        assert response.status_code == 200
+        payload = json.loads(response.content.decode("utf8"))
+        processing_configs = payload["processing_configurations"]
+        assert len(processing_configs) == 2
+        expected_names = ["default", "automated"]
+        assert all([a == b for a, b in zip(processing_configs, expected_names)])
```

Comment:
> There may be other approaches you can think of.

Maybe mocking `os.listdir` just for this test to ensure you always get the same file names?

Reply:
Sorting the lists both in the view and the test (in commit 407798d) should solve this issue.

Reply:
The reason I prefer mocking to guarantee results is that the test won't break or need updating when/if we add a new standard processing configuration (default, automated, somethingnew).

Reply:
I'm happy to change it if you'd like, but this test shouldn't break if we add a new standard processing configuration in AM. It's kind of being mocked already. See the `setUp` method for the test class:

```
def setUp(self):
        self.client = Client()
        self.client.login(username="test", password="test")
        helpers.set_setting("dashboard_uuid", "test-uuid")
        settings.SHARED_DIRECTORY = tempfile.gettempdir()
        self.config_path = os.path.join(
            settings.SHARED_DIRECTORY,
            "sharedMicroServiceTasksConfigs/processingMCPConfigs/",
        )
        if not os.path.exists(self.config_path):
            os.makedirs(self.config_path)
        install_builtin_config("default")
        install_builtin_config("automated")
```
Since the test is using a tempdir as the sharedDirectory, and the "default" and "automated" configs are manually added using the `install_builtin_config` function from archivematicaCommon's `processing` module, even if we add new built-in processing configs moving forward, the test won't be affected unless we specifically modify that `setUp` method.

I added the `install_builtin_config("automated")` to accommodate this test but otherwise the `setup` method is unchanged from what was there before.

Reply:
Ah! Thanks Tim. Sorry that I didn't catch the old behaviour from the diff. That should cover my concern then :+1: